backend-lib/build.gradle.kts

```diff
@@ -84,6 +84,9 @@ cargo {
     )
     profile = "release"
     prebuiltToolchains = true
+    exec = { spec, _ ->
```

Comment:
Tells the compiler to use given page size

please refer to https://github.com/mozilla/rust-android-gradle/blob/c62655cae09dc7b7540e96a07d35881f18ce4251/plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt#L231C34-L231C65